src/sentry/incidents/endpoints/project_alert_rule_index.py

```diff
@@ -0,0 +1,30 @@
+from __future__ import absolute_import
+
+from rest_framework import status
+from rest_framework.response import Response
+
+from sentry import features
+from sentry.api.bases.project import ProjectEndpoint
+from sentry.api.exceptions import ResourceDoesNotExist
+from sentry.incidents.endpoints.serializers import AlertRuleSerializer
+
+
+class ProjectAlertRuleIndexEndpoint(ProjectEndpoint):
+    def post(self, request, project):
+        """
+        Create an alert rule
+        """
+        if not features.has('organizations:incidents', project.organization, actor=request.user):
+            raise ResourceDoesNotExist
+
+        serializer = AlertRuleSerializer(
+            context={'project': project},
+            data=request.data,
+        )
```

Comment:
Do project rules require different permissions than projects?

Reply:
I had a look at the permissions we use for group alerts and they use the default `ProjectPermissions`, so I just followed suit.